offchain-modules/cli/src/commands/dapp/mod.rs

```diff
@@ -22,8 +24,15 @@ async fn indexer(_args: IndexerArgs) -> Result<()> {
     Ok(())
 }
 
-async fn ckb_tx_relay(_args: CkbTxRelayerArgs) -> Result<()> {
-    // TODO
+async fn ckb_tx_relay(args: CkbTxRelayerArgs) -> Result<()> {
+    let mut ckb_tx_relay = CkbTxRelay::new(
+        args.config_path,
+        args.network,
+        args.db_args,
+        args.private_key_path,
+    )
+    .await?;
+    ckb_tx_relay.start().await?;
```

Comment:
you can remove next line: `Ok(())`
```suggestion
    ckb_tx_relay.start().await
```